scalyr/dataset-addon-for-splunk
# DPDV-2719: Make sure that Inputs page works

Jira Link: <https://sentinelone.atlassian.net/browse/DPDV-2719>

# 🥅 Goal

As part of POC we are going through the Inputs page. Lets make sure that it works as expected.

# 🛠️ Solution

To make it work, I have to fix following things:

1.  Import everywhere - `import import_declare_test` - that was removed in the past, but added into only single file
2. Fix  `metadata/default.conf`- original version was blocking access of the search app to the account and therefore it was not possible to use inputs in the search application
3. Fix reading of version files - after fixing the build we haven't adjusted paths to the version files

# 🏫 Testing

* Run input tests - ` npx playwright test  --trace on --headed e2e/inputs.spec.ts`
* Check that validation procedure does not produce any errors
* Manually click through the application
* Check that versions are correctly populated

<img width="856" alt="Screenshot 2023-08-22 at 14 41 05" src="https://github.com/scalyr/dataset-addon-for-splunk/assets/122797378/408e0522-ed92-4db6-aeb0-cb9a28e488da">
<img width="950" alt="Screenshot 2023-08-22 at 14 41 42" src="https://github.com/scalyr/dataset-addon-for-splunk/assets/122797378/b9dba571-b259-4309-8474-76dbc89833c2">
<img width="1479" alt="Screenshot 2023-08-22 at 14 39 02" src="https://github.com/scalyr/dataset-addon-for-splunk/assets/122797378/c56c0a27-ee20-4b53-9950-b148c3326f4d">
